docs/content/develop/diffs.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Fix diffs"
-weight: 60
```

Comment:
nit: Generally speaking, the reason that the pages use weights separated by tens is so that (hypothetically) you don't need to reorder all pages just to add one new page. It would be nice to preserve that (here & in the rest of the pages) but I wouldn't block this PR on it.